vulnfeeds/vulns/vulns.go

```diff
@@ -364,36 +368,39 @@ func (affected *Affected) AttachExtractedVersionInfo(version cves.VersionInfo) {
 		affected.Ranges = append(affected.Ranges, gitRange)
 	}
 
-	versionRange := AffectedRange{
-		Type: "ECOSYSTEM",
-	}
-	seenIntroduced := map[string]bool{}
-	seenFixed := map[string]bool{}
-
-	for _, v := range version.AffectedVersions {
-		var introduced string
-		if v.Introduced == "" {
-			introduced = "0"
-		} else {
-			introduced = v.Introduced
+	// Adding an ECOSYSTEM version range only makes sense if we have package information.
+	if affected.Package != nil {
```

Comment:
nit: can we invert this condition and continue instead? This would avoid a lot of indentation and make the code a bit easier to read.

Reply:
Done in [ec3cf43](https://github.com/google/osv.dev/pull/845/commits/ec3cf43d42d8af6018a528f1440999b467b3f388)